jet_tracking/mpi_scripts/mpi_configs/default_config.yml

```diff
@@ -0,0 +1,23 @@
+api_msg:
```

Comment:
Could this file be moved up to higher level, as it doesn't seem to be specific to mpi? Seems kinda buried at the moment esp with nothing else in `mpi_scripts`